moveit_core/collision_detection_fcl/CMakeLists.txt

```diff
@@ -10,7 +10,12 @@ set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_V
 target_link_libraries(${MOVEIT_LIB_NAME} moveit_collision_detection ${catkin_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${LIBFCL_LIBRARIES} ${Boost_LIBRARIES})
 add_dependencies(${MOVEIT_LIB_NAME} ${catkin_EXPORTED_TARGETS})
 
-install(TARGETS ${MOVEIT_LIB_NAME}
+add_library(collision_detector_fcl_plugin src/collision_detector_fcl_plugin_loader.cpp)
+set_target_properties(collision_detector_fcl_plugin PROPERTIES VERSION ${${PROJECT_NAME}_VERSION})
+target_link_libraries(collision_detector_fcl_plugin ${catkin_LIBRARIES} ${MOVEIT_LIB_NAME})
+
+
+install(TARGETS ${MOVEIT_LIB_NAME} collision_detector_fcl_plugin
```

Comment:
Your cmakelist is missing the install command for the plugin xml.  See [here](https://github.com/ros-industrial/industrial_moveit/blob/melodic-devel/industrial_collision_detection/CMakeLists.txt#L68) for an example

Reply:
Thanks for the tip, added.